src/assets/style.css

```diff
@@ -63,3 +63,45 @@ nav.navbar-inverse .navbar-text {
 .sequence-diagram-hand, .sequence-diagram-simple {
   text-align: center;
 }
+
+/* Table of Contents */
+ul.toc {
+  padding-left: 0;
+}
+
+ul.toc li {
+  list-style-type: none;
+}
+
+ul.toc li span.active {
+  font-weight: 700;
+}
+
+ul.toc li a {
+  text-decoration: none;
+  display: block;
+}
+
+ul.toc li a:hover {
+  background-color: #eee;
+}
+
+ul.toc li ul {
+  color: #777;
+  margin-bottom: 1em;
```

Comment:
in fact, setting it to 0em still leaves quite a bit of space under each UL, but at least tightens up the space between LIs..